config/alphagov.yml

```diff
@@ -241,6 +248,40 @@ govuk-green-team:
     - 'Remember to read or add entries to the monthly newsletter (pinned to the channel) and find out what the team has been doing lately! :newspaper:'
     - 'Remember to <https://forms.gle/DdzRiptNuYQqA1Xn9|let me know> if there’s anything else I can help with or if you have any feedback. :mega:'
 
+govuk-navigation-and-presentation:
+  channel: '#navigation-and-presentation-govuk'
+  compact: true
+  exclude_titles:
+    - DNM
+    - DO NOT MERGE
+    - DRAFT
+    - PARKED
+    - PROTOTYPE
+    - WIP
+  repos:
```

Comment:
Rather than duplicating the lists of repos, it would be better to change the team owner in the [Developer Docs](https://docs.publishing.service.gov.uk/repos.html#repos-by-team) (to prevent these repo lists falling out of sync).

You can make those changes here: https://github.com/alphagov/govuk-developer-docs/blob/main/data/repos.yml

Then the Seal should work out of the box 🤞

Reply:
Nice one 🌟 , I thought it'll be more dev work to keep them in sync.

The changes are already in the repos.yml. I updated the team names in this PR so they match exactly (minor OCD trigger how the team names don’t follow the `govuk` prefix convention 😆). Is this how it magically works? I can see that it's coming from the [GITHUB_REPOS env var](https://github.com/alphagov/seal/blob/main/lib/team_builder.rb#L45)

Reply:
That should do it! 👍